design/baremetal-operator/bmh_non-bootable_iso.md

```diff
@@ -0,0 +1,113 @@
+# Add non-bootable iso attach API to BareMetalHost
+
+## Status
+
+Implementable
+
+## Summary
+
+Add a new field in BareMetalHost so that it is possible to attach a generic
+non-bootable iso image via Ironic, after the host has been provisioned.
```

Comment:
nit: I suspect we all understand what "ISO" means (it's all upper-case, by the way), but maybe we refer to "an ISO 9660 image" or "a CD (ISO 9660) image".

Reply:
Done